libmscore/shadownote.cpp

```diff
@@ -19,248 +19,264 @@
-      }
+{
+    return _notehead != SymId::noSym;
+}
 
 void ShadowNote::setState(SymId noteSymbol, int voice, TDuration duration, bool rest)
-      {
-      // clear symbols
-      _notehead = SymId::noSym;
+{
+    // clear symbols
+    _notehead = SymId::noSym;
 
-      _notehead = noteSymbol;
-      _duration = duration;
-      _voice    = voice;
-      _rest     = rest;
-      }
+    _notehead = noteSymbol;
+    _duration = duration;
+    _voice = voice;
+    _rest = rest;
+}
 
 SymId ShadowNote::getNoteFlag() const
-      {
-      SymId flag = SymId::noSym;
-      if (_rest)
-            return flag;
-      TDuration::DurationType type = _duration.type();
-      switch (type) {
-            case TDuration::DurationType::V_LONG:
-                  flag = SymId::lastSym;
-                  break;
-            case TDuration::DurationType::V_BREVE:
-                  flag = SymId::noSym;
-                  break;
-            case TDuration::DurationType::V_WHOLE:
-                  flag = SymId::noSym;
-                  break;
-            case TDuration::DurationType::V_HALF:
-                  flag = SymId::lastSym;
-                  break;
-            case TDuration::DurationType::V_QUARTER:
-                  flag = SymId::lastSym;
-                  break;
-            case TDuration::DurationType::V_EIGHTH:
-                  flag = computeUp() ? SymId::flag8thUp : SymId::flag8thDown;
-                  break;
-            case TDuration::DurationType::V_16TH:
-                  flag = computeUp() ? SymId::flag16thUp : SymId::flag16thDown;
-                  break;
-            case TDuration::DurationType::V_32ND:
-                  flag = computeUp() ? SymId::flag32ndUp : SymId::flag32ndDown;
-                  break;
-            case TDuration::DurationType::V_64TH:
-                  flag = computeUp() ? SymId::flag64thUp : SymId::flag64thDown;
-                  break;
-            case TDuration::DurationType::V_128TH:
-                  flag = computeUp() ? SymId::flag128thUp : SymId::flag128thDown;
-                  break;
-            case TDuration::DurationType::V_256TH:
-                  flag = computeUp() ? SymId::flag256thUp : SymId::flag256thDown;
-                  break;
-            case TDuration::DurationType::V_512TH:
-                  flag = computeUp() ? SymId::flag512thUp : SymId::flag512thDown;
-                  break;
-            case TDuration::DurationType::V_1024TH:
-                  flag = computeUp() ? SymId::flag1024thUp : SymId::flag1024thDown;
-                  break;
-            default:
-                  flag = SymId::noSym;
-            }
-      return flag;
-      }
+{
+    SymId flag = SymId::noSym;
+    if (_rest) {
+        return flag;
+    }
+    TDuration::DurationType type = _duration.type();
+    switch (type) {
+    case TDuration::DurationType::V_LONG: {
+        flag = SymId::lastSym;
+    }
+    break;
+    case TDuration::DurationType::V_BREVE: {
```

Comment:
this style is pretty bad, the cases don#t stick out anymore. Not mentioned in the new coding style guildlines...

Reply:
Agree with the indentation of cases.

Reply:
First, I absolutely agree that the `break;` position is wrong.
Second, we discussed it a lot. Default Qt style uses such indents. Not the ones for `break;` for sure, but for the `case`. It looks unfamiliar for me personally, but this is just a habit. The code looks pretty good and clean when the `cases` are aligned with the `switch` (checkout the new inspector code in @vpereverzev 's repo).

Reply:
Aside from the indentation, I think the braces around each case (and closing _before_ `break`?) also make it harder to read.

Reply:
The break; position is absolutely wrong. It must be placed inside the brackets with the 4-indents.

Reply:
I agree with @IsaacWeiss , the braces in every case seems overdone too me. It looks even very alien too me, never saw that before for every case, only when block was really required (e.g. when local variables were needed).

Reply:
I removed the forced addition of brackets, it remains as it was. Now the switch/case style is fully consistent with the Qt style

Reply:
good, looks better now, almost like before, just 4 space indent